quoter.lua

```diff
@@ -18,6 +21,18 @@ function preRune(bp, r)
 	if bp.Cursor:HasSelection() == false then
 		return true
 	end
+	
+	-- try using user settings, or use "default"
+	local mode = bp.Buf.Settings["quoter.mode"]
+	local quotePairs = modes[mode]
+
+	if not quotePairs then
+		micro.InfoBar():Error("Unknown quoter mode: \"" .. mode .. "\". Appliying mode \"default\".")
+		-- config.SetGlobalOption("quoter.mode", "default")
+		mode = "default"
+		quotePairs = modes["default"]
+	end
```

Comment:
There is a typo and I realized that it wouldn't probably be good if an error is always displayed, so I think these changes can be done:
```suggestion
	if not quotePairs then
		bp.Buf:SetOption("quoter.mode", "default")
		quotePairs = modes["default"]
		micro.InfoBar():Message("Unknown quoter mode: \"" .. mode .. "\". Mode is set to \"default\".")
	end
```

Reply:
Actually, using a different mode (`default`) is not expected so it's probably better to instead only display an error about changing it (like in `settings.json` then running `reload`). The plugin may be expected to be used when there is a selection, so the callback could also return `false` to not insert the character.